### ISSUE TITLE
Accuracy Problem2

### ISSUE THREAD

**Issue by dedoogong** [yysijie/st-gcn#151]
### Code version (Git Hash) and PyTorch version
979ae91 / pytorch 1.0.1
### Dataset used
NTU-RGB-D 
### Expected behavior
around 88% accuracy
### Actual behavior
68~70% accuracy
### Steps to reproduce the behavior
1. run ntu_gendata.py --data_path /media/lee/ssd/nturgb+d_skeletons
    > I got 4 npy,pkl files(987.5MB for training, 511.9MB for eval).
2. and then run main.py recognition -c config/st_gcn/ntu-xview/train.yaml
### Other comments

Dear @yysijie, I'm trying to improve your model and testing it.
After clone this repo(new stgcn version), I'm using 2 GPUs (2x1080 == 2x8GB) and just set the batch size to 32 instead of 64 and device to [0,1]
after several trials, I just only see always the mAP of around 68~70.
I can't reproduce even the same result as the original paper.

Do I need to set "random_choose/random_move" when I generate the dataset?

Now, I'm trying to use 4 or 8 GPUs (4~8x16GB) again and use the same default hyper parameters, but the result stilll seems similar(even little bit worse).

By the way, when I evaluate the provided NTURGB model, the mAP is right(89%).

Hum anybody can guess the reason??

Thank you!

**Comment by dedoogong** [comment]
is it related to the unbalanced distribution of NTU-RGB-D? If so, what should I do for managing the balance? Please give me a hint.
